src/common/layout/Header.js

```diff
@@ -201,6 +201,15 @@ const renderLoggedOut = (
   );
 };
 
+type _SyntheticInputEvent = {
+  target: { // eslint-disable-line
```

Comment:
always opt to disable specific rule for line, not all

Reply:
:+1: